packages/create-app/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@noaignite/create-app",
+  "version": "0.5.0",
+  "private": false,
+  "author": "NoA Ignite",
+  "description": "Create NoA Ignite react apps with one command",
+  "keywords": [
+    "react"
+  ],
+  "license": "MIT",
+  "bin": {
+    "create-app": "./index.js"
+  },
+  "files": [
+    "index.js",
+    "createApp.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/noaignite/accelerator.git",
+    "directory": "packages/create-app"
+  },
+  "bugs": {
+    "url": "https://github.com/noaignite/accelerator/issues"
+  },
+  "scripts": {
+    "build": "pnpm prebuild && mkdir dist && cp -a package.json LICENSE CHANGELOG.md README.md createApp.js index.js dist/",
+    "prebuild": "rm -rf dist",
+    "release": "pnpm build && npm publish ./dist --tag latest",
+    "release:yalc": "pnpm build && yalc publish ./dist"
+  },
```

Comment:
**thought:** Remove these

Since we don't use the `dist` directory at all. We simply target the file to execute by applying `./index.js` to the `bin.createApp` property.